frontend/messages/en.json

```
@@ -1196,6 +1196,39 @@
+	"riskLevelHelpText": "No risk: breach tracking required.\nRisk: breach tracking required and authorities must be notified.\nHigh risk: breach tracking required, authorities and data subjects must be notified.",
+	"affectedSubjectsCount": "Affected data subjects count",
+	"affectedProcessings": "Affected processings",
+	"affectedPersonalData": "Affected personal data",
+	"affectedPersonalDataCount": "Affected personal data count",
+	"authorities": "Authorities",
+	"authorityNotifiedOn": "Authority notified on",
+	"authorityNotificationRef": "Authority notification reference",
+	"subjectsNotifiedOn": "Data subjects notified on",
+	"potentialConsequences": "Potential consequences",
+	"remediationMeasures": "Remediation measures",
+	"incident": "Incident",
```

Biome (2.1.2)

[error] The key incident was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)